packages/libs/react-ui/src/components/Card/Card.css.ts

```diff
@@ -18,7 +18,9 @@ export const container = style([
   }),
 ]);
 
-export const fullWidthClass = style([sprinkles({ width: '100%' })]);
+export const fullWidthClass = style({
```

Comment:
if I dont, the width is overwritten by the "max-content" set earlier

Reply:
I think i'm missing some context. Why was this switched from a sprinkle to custom styling?

Reply:
for some reason when using the sprinkels its specificity was to low.
removing it, it changes in the current classname. overwriting the old value